ipatests/test_integration/test_pwpolicy.py

```diff
@@ -261,3 +261,35 @@ def test_minlength_add(self):
         )
         assert result.returncode != 0
         assert 'minlength' in result.stderr_text
+
+    def test_maxlife(self):
+        """Check maxlife error message where minlife > maxlife specified
```

Comment:
If its checking only error text then i would suggest you to add this test 
in test_xmlrpc/test_pwpolicy_plugin.py

Reply:
+1